src/impl.cpp

```diff
@@ -403,6 +475,8 @@ void Manifold::Impl::CreateHalfedges(const Vec<ivec3>& triProp,
   const auto body = [&](int i, int consecutiveStart, int segmentEnd) {
     const int pair0 = ids[i];
     Halfedge& h0 = halfedge_[pair0];
+    const int startVert = h0.startVert;
+    const int endVert = h0.endVert;
```

Comment:
is this change necessary?

Reply:
no, it was for something else before and I found it doesn't work